prdoc/pr_2949-async-backing-on-all-testnet-system-chains.prdoc

```diff
@@ -0,0 +1,21 @@
+title: Enable async backing on all testnet system chains
+
+doc:
+  - audience: Runtime User
+    description: |
+      Async backing has been enabled on all testnet system chains: asset-hub-westend,
+      bridge-hub-westend, bridge-hub-rococo, collectives-westend, contracts-rococo,
+      coretime-westend, coretime-rococo, people-westend, and people-rococo. These now target 6s
+      block times.
```

Comment:
On coretime at least, this will change the configuration required to achieve the same sale `leadin_length` and `interlude_length` durations in real time (they should both double). I think this should be included in the changelog, as for running chains the configuration will need to be updated after the runtime upgrade but before the end of the current region.

Reply:
Sorry missed the notification - see here:
https://github.com/paritytech/polkadot-sdk/blob/452f81366870436634f72e2a187179e5a9ad9894/substrate/frame/broker/src/types.rs#L261

In the configuration these two types are the parachain block number, whereas the `advance_notice` is the relay chain block number.

When we set the config on Rococo it was set with 12s block times in mind for these two values. I wonder does it make more sense to just change these to be the relay block number as well

Reply:
Ahh okay!

Reply:
We should rewrite this to also use the relay chain block number.